lib/pages/movies_page.dart

```diff
@@ -0,0 +1,180 @@
+import 'package:flutter/material.dart';
+import '../../../pages/details_page.dart';
+import '../../../utils/i_movie_repository.dart';
+import '../../../widgets/movie_poster.dart';
+import '../../../widgets/movie_score.dart';
+import '../../../utils/ui_constants.dart';
+import '../../../models/movie.dart';
+import '../../../models/movie_model.dart';
+
+class MoviesPage extends StatefulWidget {
+  const MoviesPage({
+    super.key,
+  });
+
+  static const double borderRadius = 4;
+  static const double rowPadding = 15;
+  static const double posterHeight = 150 * 1.3;
+  static const double posterWidth = 100 * 1.3;
+  static const double imageSpacing = 20;
+  static const double textSpacing = 5;
+  static const double titleFontSize = 20;
+  static const double scoreIconScale = 0.7;
+  static const int displayedGenresAmount = 3;
+  static const double genreCardPadding = 2;
+  static const Text noDataText = Text('No data');
+
+  @override
+  State<MoviesPage> createState() => _MoviesPageState();
+}
+
+class _MoviesPageState extends State<MoviesPage> {
+  late Future<List<Movie>?> movies;
+
+  @override
+  void initState() {
+    super.initState();
+    movies = IMovieRepository().getMovies();
+  }
+
+  Card _buildMovieCard(Movie movie) {
+    String releaseYear = DateTime.parse(movie.releaseDate).year.toString();
+
+    return Card(
+      color: Colors.blueGrey[800],
```

Comment:
You can change this value to Colors.blueGrey.shade800

Reply:
done